src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -274,44 +290,17 @@ private void filmSuchen1(String strUrlFeed, String thema, boolean weiter) {
                         urlSendung = urlSendung.replace("&amp;", "&");
                     }
 
-                    filmSuchen2(url, thema, titel, d, datum, zeit, urlSendung);
-                }
-            }
-            if (!Config.getStop() && weiter
-                    && (CrawlerTool.loadLongMax() || thema.equalsIgnoreCase("alpha-Centauri"))) {
-                // dann gehts weiter
-                int maxWeiter = 0;
-                int maxTh = 10;
-                String urlWeiter = strUrlFeed + "&mcontents=page.";
-                for (int i = 2; i < maxTh; ++i) {
-                    ///tv/Abendschau/Sendung?documentId=14913430&amp;bcastId=14913430&amp;mcontents=page.2"
-                    if (seite1.indexOf("&amp;mcontents=page." + i) != -1) {
-                        maxWeiter = i;
-                    } else {
-                        break;
-                    }
-                }
-                for (int i = 2; i < maxTh; ++i) {
-                    if (Config.getStop()) {
-                        break;
-                    }
-                    if (i <= maxWeiter) {
-                        filmSuchen1(urlWeiter + i, thema, false);
-                    } else {
-                        break;
-                    }
-
-                }
-            }
-        }
+				}
+			}
+		}
 
         private void filmSuchen2(String urlFilm_, String thema, String titel, long dauer, String datum, String zeit, String urlSendung) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 58 to the 15 allowed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3776)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)